src/libs/TransactionUtils.js

```diff
@@ -62,8 +64,8 @@ function buildOptimisticTransaction(
         currency,
         reportID,
         comment: commentJSON,
-        merchant,
-        created: DateUtils.getDBTime(),
+        merchant: merchant || CONST.REPORT.TYPE.IOU,
+        created: created || DateUtils.getDBTime(),
```

Comment:
> created: created
This won't work, right?

I think the server is expecting time in one format. i.e. YYYY-MM-DD
```suggestion
        created: DateUtils.getDBTime(created) || DateUtils.getDBTime(),
```

Reply:
The server does not mind, it can be both, now its passing `DateUtils.getDBTime()` and it should work fine